examples/Remarketing/AddUnionUserList.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Ads\GoogleAds\Examples\Remarketing;
+
+require __DIR__ . '/../../vendor/autoload.php';
+
+use GetOpt\GetOpt;
+use Google\Ads\GoogleAds\Examples\Utils\ArgumentNames;
+use Google\Ads\GoogleAds\Examples\Utils\ArgumentParser;
+use Google\Ads\GoogleAds\Lib\V4\GoogleAdsClient;
+use Google\Ads\GoogleAds\Lib\V4\GoogleAdsClientBuilder;
+use Google\Ads\GoogleAds\Lib\V4\GoogleAdsException;
+use Google\Ads\GoogleAds\Lib\OAuth2TokenBuilder;
+use Google\Ads\GoogleAds\Util\V4\ResourceNames;
+use Google\Ads\GoogleAds\V4\Common\LogicalUserListInfo;
+use Google\Ads\GoogleAds\V4\Common\LogicalUserListOperandInfo;
+use Google\Ads\GoogleAds\V4\Common\UserListLogicalRuleInfo;
+use Google\Ads\GoogleAds\V4\Enums\UserListLogicalRuleOperatorEnum\UserListLogicalRuleOperator;
+use Google\Ads\GoogleAds\V4\Errors\GoogleAdsError;
+use Google\Ads\GoogleAds\V4\Resources\UserList;
+use Google\Ads\GoogleAds\V4\Services\UserListOperation;
+use Google\ApiCore\ApiException;
+use Google\Protobuf\StringValue;
+
+/**
+ * Creates a combination user list containing users that are present on any one of the provided user
+ * lists.
+ */
+class AddUnionUserList
```

Comment:
I wonder why we name this "UnionUserList".
Nowhere in the code that has the word "union". 
Instead, we call this kind of user list "combination user list" in many places.

Reply:
@devchas any background you could provide on this as you wrote the golden?

Reply:
Please see the [conversation history](https://github.com/googleads/google-ads-java/pull/241) on the PR - specifically comments made by Nick on Mar 3.

Reply:
Thanks @devchas for the link. SGTM, any strong feelings about this @fiboknacky ?

Reply:
@fiboknacky Is it OK with you if I keep it like that?

Reply:
If Devin doesn't object to this, I prefer we use the name CombinationUserList instead.
We use that word in so many places throughout the code, so why do we just name the example differently?

Reply:
I am okay with changing the name of the example. However, I think AddCombinationUserList is too similar to AddCombinedRuleUserList and may cause confusion. Part of the reason we ended up with the name AddUnionUserList is that I wrote several UserList-related examples all at the same time for a guide, and we wanted to make sure that all of the names were different enough as not to cause confusion.

Reply:
What about "AddLogicalUserList" instead?

Reply:
Okay with me.

Reply:
@fiboknacky If OK with you, I will change to AddLogicalUserList as it seems to be the best tradeoff.